File: app/src/main/java/tw/y_studio/ptt/ui/BaseActivity.java
package tw.y_studio.ptt.ui;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;

public class BaseActivity extends AppCompatActivity {

    public void closeAllFragment() {}

    public void loadFragment(Fragment toFragment, Fragment thisFragment) throws Exception {}

    public void loadFragmentNoAnim(Fragment toFragment, Fragment thisFragment) throws Exception {}
}

File: app/src/main/java/cc/ptt/android/base/BaseActivity.kt
package cc.ptt.android.base

import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.Fragment

open class BaseActivity : AppCompatActivity() {
    open fun closeAllFragment() {}

    @Throws(Exception::class)
    open fun loadFragment(toFragment: Fragment?, thisFragment: Fragment?) {
    }

    @Throws(Exception::class)
    open fun loadFragmentNoAnim(toFragment: Fragment?, thisFragment: Fragment?) {
    }
}
